files/en-us/web/html/element/pre/index.html

```diff
@@ -78,6 +78,8 @@ <h3 id="Result">Result</h3>
 
 <p>{{EmbedLiveSample("Example")}}</p>
 
+<div class="notecard note"><strong>Note:</strong> If you have HTML tags in the content you want to display literally, rather than parsed as HTML, you will need to encode the angle brackets. For example <code>&amp;lt;p&amp;gt;</code> to display <code>&lt;p&gt;</code>.</div>
```

Comment:
```suggestion
<div class="notecard note"><p><strong>Note:</strong> If you have HTML tags in the content you want to display literally, rather than parsed as HTML, you will need to encode the angle brackets. For example <code>&amp;lt;p&amp;gt;</code> to display <code>&lt;p&gt;</code>.</p></div>
```

(to keep the Markdown converter happy)